xml/System.Security.Cryptography/AsymmetricAlgorithm.xml

```diff
@@ -459,11 +459,27 @@
         <Parameter Name="pbeParameters" Type="System.Security.Cryptography.PbeParameters" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="passwordBytes">To be added.</param>
-        <param name="pbeParameters">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="passwordBytes">The bytes to use as a password when encrypting the key material.</param>
+        <param name="pbeParameters">The password-based encryption (PBE) parameters to use when encrypting the key material.</param>
+        <summary>Exports the current key in the PKCS#8 EncryptedPrivateKeyInfo format.</summary>
```

Comment:
```suggestion
        <summary>Exports the current key in the PKCS#8 EncryptedPrivateKeyInfo format with a read-only byte span containing the password.</summary>
```
It's best to disambiguate overloads in the description. This is just one suggestion.

Reply:
Thoughts on `Exports the current key in the PKCS#8 EncryptedPrivateKeyInfo format with a byte-based password.`? (and char-based for the overload)

Reply:
That sounds fine, @bartonjs.